src/components/shared/header/index.js

```diff
@@ -4,6 +4,7 @@ import Link from '../../ui/link'
 import SiteNav from './site-nav'
 import UserNav from './user-nav'
 import Hexagons from './hexagons.svg'
+import getModalName from '../../../services/modal.service.js'
```

Comment:
This is not how services work. Check to learn about them https://github.com/eyedea-io/zefir#services